app/reports/service_requests.rb

```diff
@@ -99,6 +100,10 @@ def column_attrs
       end
     end
 
+    if params[:fulfillment_info]
+      attrs["Sent to SPARCFulfillment"] = "service_request.try(:protocol).try(:sub_service_request).try(:in_work_fulfillment?) ? 'Yes' : 'No'"
```

Comment:
That's what I would do (and no `try`'s to boot), but @jleonardw9 should double check that's the right condition.